common/djangoapps/util/module_utils.py

```diff
@@ -3,7 +3,7 @@
 """
 
 
-def yield_dynamic_descriptor_descendents(descriptor, module_creator):  # pylint: disable=invalid-name
+def yield_dynamic_descriptor_descendents(descriptor, user_id, module_creator):  # pylint: disable=invalid-name
```

Comment:
Typo in the word "descendents", should be "descendants". Descendents is a band. :)